### DISCUSSION TITLE
Ideas

### DISCUSSION THREAD

**Discussion by lokihiddleston** [AprilSylph/XKit-Rewritten#779] [Suggestion Box]
Hey! ^^

I thought, would it be possible to have an option to separate the formats by categories in our "favorites". Like everything that is "photos" / "quotes" / "videos" / ...

I find that it would be easier to sort things and find them in our favorites by categories.

**Comment by JRHB314** [comment]
by favorites do you mean likes?

**Comment by lokihiddleston** [comment]
Yes :)
the posts we like, they go in "likes" area right? well in this "likes" place, If we could sort them by categories:

- texts
- images
- quotes
- videos
- ...
